### DISCUSSION TITLE
svn to git

### DISCUSSION THREAD

**Discussion by lolzim** [markcheret/footnotes#115] [General]
Hi,

I usually work with svn, so I hope I wouldn't make something wrong. If so, please tell me quickly that & what I made wrong.

Beside I knew that one day I would have to cross the street... Knowlegde is power!
🙃

**Comment by Rumperuu** [comment]
I'll repost some of the tips and guides I sent over email in the Wiki [here](https://github.com/markcheret/footnotes/wiki/Git-Tips) - gimmie a sec.

**Reply by Rumperuu** [reply]
Done!

**Comment by lolzim** [comment]
Thank you! I'll check this tonight and go ahead on #103

**Reply by lolzim** [reply]
OK, I made just the branch & pushed the fix into it (Mainly following Ben's "Workflow" wiki checklist.

**Comment by lolzim** [comment]
I found a "universal" solution for bash completion here:
https://stackoverflow.com/questions/12399002/how-to-configure-git-bash-command-line-completion

Maybe you can add extract of it into wiki; I let you see if this could be of any help.

**Reply by Rumperuu** [reply]
I've added the link (and also a couple more useful tips for working with branches - `git branch` and `git branch -a`)

**Reply by Rumperuu** [reply]
Added another really useful trick (that I actually only recently learnt) - `git add -p`
